src/api/routes/rosetta/mempool.ts

```diff
@@ -30,7 +30,11 @@ export function createRosettaMempoolRouter(db: DataStore, chainId: ChainID): Rou
       return;
     }
 
-    const { results: txResults } = await db.getMempoolTxIdList();
+    const { results: txResults } = await db.getMempoolTxList({
+      limit: Number.MAX_SAFE_INTEGER,
```

Comment:
🤔

Reply:
This is the appropriate reaction lol, I replied here: https://github.com/blockstack/stacks-blockchain-api/pull/629/files#r664607180